# HG changeset patch
# User Rik <rik@octave.org>
# Date 1379615235 25200
#      Thu Sep 19 11:27:15 2013 -0700
# Node ID 4e86bb1901d6e048d9f58c98b2f70fc0f8a8c98f
# Parent  c36d3a4bc42f123cc3c9c75a7c02d2aaf0c6e379
caxis.m: Add more information to docstring.

* scripts/plot/caxis.m: Add more information to docstring.

diff --git a/scripts/plot/caxis.m b/scripts/plot/caxis.m
--- a/scripts/plot/caxis.m
+++ b/scripts/plot/caxis.m
@@ -12,31 +12,33 @@
 ## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
-## @deftypefn  {Function File} {} caxis (@var{limits})
+## @deftypefn  {Function File} {} caxis ([cmin cmax])
 ## @deftypefnx {Function File} {} caxis ("auto")
 ## @deftypefnx {Function File} {} caxis ("manual")
 ## @deftypefnx {Function File} {} caxis (@var{hax}, @dots{})
 ## @deftypefnx {Function File} {@var{limits} =} caxis ()
 ## Query or set color axis limits for plots.
 ##
-## The argument @var{limits} should be a 2-element vector specifying the
+## The limits argument should be a 2-element vector specifying the
 ## lower and upper limits to assign to the first and last value in the
-## colormap.  Values outside this range are clamped to the first and last
+## colormap.  Data values outside this range are clamped to the first and last
 ## colormap entries.
 ##
-## If @var{limits} is @qcode{"auto"}, then automatic colormap scaling is
-## applied, whereas if @var{limits} is @qcode{"manual"} the colormap scaling
-## is set to manual.
+## If the @qcode{"auto"} option is given then automatic colormap limits are
+## applied.  The automatic algorithm sets @var{cmin} to the minimum data value
+## and @var{cmax} to the maximum data value.  If @qcode{"manual"} is specified
+## then the @qcode{"climmode"} property is set to @qcode{"manual"} and the
+## numeric values in the @qcode{"clim"} property are used for limits.
 ##
 ## If the first argument @var{hax} is an axes handle, then operate on
 ## this axis rather than the current axes returned by @code{gca}.
 ##
 ## Called without arguments the current color axis limits are returned.
 ## @seealso{colormap}
 ## @end deftypefn
 
